dev/tools/update_icons.dart

```diff
@@ -84,6 +82,10 @@ const Map<String, String> _identifierRewrites = <String, String>{
   '23mp': 'twenty_three_mp',
   '24mp': 'twenty_four_mp',
   'class': 'class_',
+  // TODO(guidezpl): will clean these up in g3
```

Comment:
?

Reply:
these are icons in g3 that for some reason had to have a different name (name clashes possibly?). After I finish the 1P side of the work, I should be able to get rid of these

Reply:
Point to a bug #, right Hixie?

Reply:
I added a bug #